ci-operator/config/stackrox/stackrox/stackrox-stackrox-master.yaml

```diff
@@ -164,6 +171,8 @@ tests:
         name: operator-bundle
       - env: OPERATOR_BUNDLE_INDEX_MAGE
         name: operator-bundle-index
+      - env: MOCK_GRPC_SERVER_IMAGE
```

Comment:
Do you want to also add this to `merge-push-and-release` so that this image is available to match the main branch tags?